db/post_migrate/20241023041126_clear_duplicate_admin_notices.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+class ClearDuplicateAdminNotices < ActiveRecord::Migration[7.1]
+  def up
+    problem_subject_id = 0
+
+    DB.exec(<<~SQL)
+      DELETE FROM admin_notices
+      WHERE subject = #{problem_subject_id}
```

Comment:
It looks ok, but could you please explain why we are searching for AdminNotice with `subject = 0`

Reply:
It's a Rails enum. Currently it only has one value `problem`, at index 0, and I don't want to reference the Rails code in the migration. 🙂

Reply:
Ah, perfect, thank you :)